misc/buildtestdata.sh

```diff
@@ -33,49 +33,8 @@ mkdir -p "${E2EDATA}"
 
 (cd "${E2EDATA}" && tar -j -c -f net_done.tar.bz2 --exclude node.log --exclude agreement.cdv net)
```

Comment:
I believe you can use `tar -C` to specify the directory in which to perform the `tar` instead of `cd`ing in a subshell.